index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Skeleton</title>
+
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo"><h2>My logo</h2></div>
+        <div class="menu"><img src="images/Enabled.png" alt="" /></div>
+      </nav>
+    </header>
+    <main>
+      <section class="hello">
+        <div class="hellosection">
+          <div class="box box1">I’m Mariane Glad to see you!</div>
+          <div class="box box2">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </div>
+          <div class="box box3">
+            <div>LET'S CONNECT</div>
+            <div>
+              <img class="social" src="images/twitter.svg" alt="" /><img class="social" src="images/Linkedin.svg" alt="" />
+              <img class="social" src="images/M.svg" alt="" /><img class="social" src="images/cat.svg" alt="" /> <img class="social" src="images/Y.svg" alt="" />
```

Comment:
- [ ] Please, consider the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

Reply:
Created list for icons and link each icon to an a tag!